src/write-archive/index.test.ts

```diff
@@ -119,7 +123,7 @@ describe('writeTestResult', () => {
       } as TestInfo,
       { home: Buffer.from('Chromatic') },
       { 'http://localhost:3000/home': { statusCode: 200, body: Buffer.from('Chromatic') } },
-      { viewport: { height: 480, width: 720 } },
+      { viewports: [720] },
```

Comment:
Do we have test coverage for all of the `ChromaticOption`s? It doesn't seem like it, and I don't think we need coverage of every single combination, but a little more coverage might be better.

Reply:
We have a few of them tested [here](https://github.com/chromaui/test-archiver/blob/b767c4e28f716bfdd4155768b7abfd67f574df7f/src/write-archive/index.test.ts#L22-L26). Should I include them all?

Reply:
Ok, no, that seems sufficient.